code/modules/world_topic/adminmsg.dm

```diff
@@ -0,0 +1,36 @@
+/datum/world_topic_handler/adminmsg
+	topic_key = "adminmsg"
+	requires_commskey = TRUE
+
+/datum/world_topic_handler/adminmsg/execute(list/input, key_valid)
+	/*
+	We got an adminmsg from the Discord bot, so lets split the input then validate the input. Expected output:
+		1. adminmsg = ckey of person the message is to
+		2. msg = contents of message, parems2list requires
```

Comment:
Pre-existing code but.. it requires what?
Also should be `params2list`, I assume

Reply:
Yeah, I'll fix that when I can